compiler/rustc_errors/src/lib.rs

```diff
@@ -661,6 +661,19 @@ impl Handler {
         inner.stashed_diagnostics = Default::default();
     }
 
+    /// Avoid failing compilation in the presence of delayed bugs.
+    ///
+    /// NOTE: *do not* call this function from rustc. It is only meant to be called from rustdoc.
+    /// Rustdoc documents multiple targets at once, meaning there may be multiple versions
+    /// of the same function in scope at the same time, which isn't legal Rust otherwise. See
+    /// https://doc.rust-lang.org/beta/rustdoc/advanced-features.html#interactions-between-platform-specific-docs
+    /// for details
+    pub fn reset_delayed_bugs(&self) {
```

Comment:
```suggestion
    pub fn reset_delayed_bugs(&self) {
assert!(self.opts.actually_rustdoc);
```

Reply:
😆 I want to remove that flag at some point